src/components/FoodBox.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import FoodItem from './FoodItem'
+
+export default class FoodBox extends Component {
+    render() {
+
+     const{items} = this.props
+
+        return (
+          <div>
+            {
+                items.map((item)=>{
+                    return <FoodItem
```

Comment:
always remember to add keys to any element inside a loop in react. This could be the index of the element taken from the parameter.